packages/gatsby/src/utils/feedback.ts

```diff
@@ -111,3 +107,39 @@ export async function userPassesFeedbackRequestHeuristic(): Promise<boolean> {
   // for feedback
   return true
 }
+
+function isFeedbackDisabled(): boolean {
+  // Heuristic 2
+  if (getConfigStore().get(feedbackKey) === true) {
+    return true
+  }
+
+  // Heuristic 3
+  if (process.env.GATSBY_FEEDBACK_DISABLED === `1`) {
```

Comment:
We have a convenience method for these: https://github.com/gatsbyjs/gatsby/blob/ecc0ad37f3fb6d542964c95c2f853c25cc99d5b8/packages/gatsby-telemetry/src/is-truthy.ts#L3
not sure if it's worth moving to core or elsewhere.

Reply:
Noted. I think in this one off case we're ok, but certainly something to be aware of moving forward.

Reply:
👍